docassemble/ALWeaver/data/questions/assembly_line.yml

```diff
@@ -608,6 +612,123 @@ fields:
 code: |
   interview.short_title = interview.title
 ---
+id: customize next steps
+question: |
+  "Next steps" instructions
+subquestion: |
+  By default, your interview will include a customizable "Next Steps"
+  document. 
+
+  This is the default uncustomized document:
+
+  ${ next_steps_documents[interview.form_type]['attachment']["preview"].pdf }
+
+  You can fully customize this file in the playground. You can
+  also make some simple customizations now.
+
+  ${ collapse_template(how_to_customize_next_steps_template) }
+fields:
+  - Include a next steps instructions document: interview.include_next_steps
+    datatype: yesno
+    default: True
+  - Customize some of the instructions now?: interview.customize_next_steps
+    datatype: yesnoradio
+    show if: interview.include_next_steps
```

Comment:
Flows a bit better if you remove some of the stuff from the subquestion and put it behind a show if.

```suggestion
  - Include a next steps instructions document?: interview.include_next_steps
    datatype: yesnoradio
    default: True
  - note: |
      You can fully customize this file in the playground. You can
      also make some simple customizations now.
      
      ${ collapse_template(how_to_customize_next_steps_template) }
    show if: interview.include_next_steps
  - Customize some of the instructions now?: interview.customize_next_steps
    datatype: yesnoradio
    show if: interview.include_next_steps
```